### PR TITLE
Feature/add fine payment

### PR DESCRIPTION
**Pull request by VitalyBashkiser**
**Add FINE Payment for book overdue**

1. If during returning Borrowing, the actual_return_date is greater than the expected_return_date - ask the user to pay fine
2. Create a FINE Payment with some preconfigured FINE_MULTIPLIER (for ex. 2)
3. Payment money_to_pay should be calculated as “days of overdue” * daily_fee * FINE_MULTIPLIER

image

### CODE REVIEW

File: library/signals.py [hunk 1]
```diff
@@ -37,3 +41,26 @@ def send_telegram_notification(sender, instance, created, **kwargs):
...
+                TELEGRAM_API_URL,
+                data={"chat_id": user.telegram_chat_id, "text": message},
+            )
+            print(response.json())
+
+
+@receiver(post_save, sender=Borrowing)
+def handle_new_borrowing(sender, instance, created, **kwargs):
+    if created:
+        async_task(send_borrowing_notification, instance, created)
```
Comment (Gilgumish) [inline library/signals.py]: This fragment can be deleted. I moved it to the borrowing/signals.py

**Review by Gilgumish — APPROVED** [review]
All is well, but delete that fragment

File: library/signals.py (hunk 1 above)
Comment (VitalyBashkiser) [reply library/signals.py]: Got it

**Review by Boyok1337 — APPROVED** [review]

**Review by Gilgumish — APPROVED** [review]
GJ

**Pull request merged**